text/3533-combine-infra-and-release-teams.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: combine-infra-and-release
+- Start Date: 2023-11-23
+- RFC PR: [rust-lang/rfcs#3533](https://github.com/rust-lang/rfcs/pull/3533)
+- Rust Issue: N/A
+
+# Summary
+
+This RFC proposes merging the Release team into the Infrastructure team as a subteam. The membership of the Infrastructure and Release teams proper remain the same.[^subteam]
+
+[^subteam]: Note: Members of subteams are not automatically members of their parent team. So Release members will be part of the Infra team family but *not* members of the team proper.
```

Comment:
```suggestion
[^subteam]: Note: Members of subteams are not automatically full members of their parent team. So Release members will be part of the Infra team family but *not* members of the team proper. For example, they Release members would not get checkbox authority for Infra, but Release members could serve as the Leadership Council representative for Infra.
```

Even though we don't have real clear terminology and definitions around here, I think it'd be good to highlight some of the practical implications of the arrangement.